README.md

```diff
@@ -246,7 +246,14 @@ After a document dies, the corresponding worklet will be kept alive for maximum
 
 ## Permissions Policy
 
-Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin). 
+Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+The sharedStorage.selectURL() method can be disallowed by the "shared-storage-select-url" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+### Permissions Policy inside the shared storage worklet
+The permissions policy inside the shared storage worklet will be identical to the permissions policy on the associated document.
```

Comment:
maybe: s/will be identical to/will inherit/

Reply:
Done.